ext/nnstreamer/tensor_filter/tensor_filter_snpe.cc

```diff
@@ -589,6 +589,10 @@ snpe_subplugin::configureUserBuffer (zdl::DlSystem::UserBufferMap &buffer_map, c
     std::vector<size_t> strides (bufferShape.rank ());
     strides[strides.size () - 1] = sizeof (float);
     for (size_t i = strides.size () - 1; i > 0; --i) {
+      /* Some model has "0" dim, replacing 0 with 1 */
+      if (bufferShape[i] == 0) {
```

Comment:
Is this the SNPE bug you've mentioned yesterday?

Reply:
let's add something like:
```
  static int reported = 0;
  if (G_UNLIKELY(!reported)) {
    reported = 1;
    log info-level (....)
  }
```

Reply:
Right, after replacing 0 with 100, it works. Now I've asked the model making counterpart.